tests/integration-tests/Apple/Device.Commands.Tests.proj

```diff
@@ -0,0 +1,45 @@
+<Project DefaultTargets="Test">
+  <Import Project="../Helix.SDK.configuration.props"/>
+
+  <ItemGroup>
+    <HelixTargetQueue Include="osx.1015.amd64.open"/>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <TestAppBundleName>HelloiOS</TestAppBundleName>
+    <XHarnessTestAppBundleUrl>https://netcorenativeassets.blob.core.windows.net/resource-packages/external/ios/test-app/ios-device/$(TestAppBundleName).app.zip</XHarnessTestAppBundleUrl>
+    <TestAppDestinationDir>$(ArtifactsTmpDir)test-app\ios-device</TestAppDestinationDir>
```

Comment:
I am 100% sure about this but I was told once that it's better to use `\` everywhere so that paths work on Windows and UNIX. But I think it doesn't matter much and MSBuild knows it should always transform them to the right ones. I think even mixing them in one string will help.

And I have seen everyone using backslashes in Arcade (or at least the code I worked with) so I keep using these.

I can be totally wrong about this though and either it doesn't matter or forward is better since Windows should handle both.

I tried googling and I didn't find much.